f8a_report/helpers/report_helper.py

```diff
@@ -64,32 +64,34 @@ def __init__(self):
 
         self.emr_api = os.getenv('EMR_API', 'http://f8a-emr-deployment:6006')
 
+    def cleanup_tables(self, table_name, env_var, num_days):
+        """Cleanup tables on a periodic basis."""
+        # Number of days to retain the celery task_meta data
```

Comment:
Remove redundant comments, you have to add comments only if the statement is doing something which is hard to understand. Comments on obvious statements should be avoided.

Applicable to all comments in your changes.